pkgs/top-level/aliases.nix

```diff
@@ -660,6 +660,7 @@ mapAliases ({
   maui-nota = libsForQt5.mauiPackages.nota; # added 2022-05-17
   mbox = throw "'mbox' has been removed, as it was broken and unmaintained"; # Added 2023-12-21
   mcomix3 = mcomix; # Added 2022-06-05
+  megapixels = Megapixels; # Added 2024-02-13
```

Comment:
We generally don't uppercase attrs as this makes it harder to discover the package and just guessing the name.